<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvPendulumWindow</class>
 <widget class="QMainWindow" name="InvPendulumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inverted Pendulum</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="angle_plot" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QCustomPlot" name="velocity_plot" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QCustomPlot" name="cost_plot" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
